pgscatalog.core/src/pgscatalog/core/__init__.py

```diff
@@ -14,56 +15,20 @@
     TargetVariants,
     TargetVariant,
     TargetType,
-    BasePGSException,
-    MatchError,
-    DuplicateMatchError,
-    MatchRateError,
-    ZeroMatchesError,
-    MatchValueError,
-    CombineError,
-    BuildError,
-    ScoreFormatError,
-    EffectTypeError,
-    CatalogError,
-    ScoreDownloadError,
-    ScoreChecksumError,
-    QueryError,
-    InvalidAccessionError,
-    SamplesheetError,
-    GenomesNotFound,
-    SamplesheetFormatError,
     RelabelArgs,
     relabel,
     relabel_write,
     effect_type_keyfunc,
     chrom_keyfunc,
     EffectType,
-    Allele,
+    models,
 )
 
 log_fmt = "%(name)s: %(asctime)s %(levelname)-8s %(message)s"
 logging.basicConfig(format=log_fmt, datefmt="%Y-%m-%d %H:%M:%S")
 logger = logging.getLogger(__name__)
 
 __all__ = [
-    "BasePGSException",
-    "MatchError",
-    "DuplicateMatchError",
-    "MatchRateError",
-    "ZeroMatchesError",
-    "MatchValueError",
-    "CombineError",
-    "BuildError",
-    "ScoreFormatError",
-    "EffectTypeError",
-    "CatalogError",
-    "ScoreDownloadError",
-    "ScoreChecksumError",
-    "QueryError",
-    "InvalidAccessionError",
-    "SamplesheetError",
-    "GenomesNotFound",
-    "SamplesheetFormatError",
```

Comment:
Removed these exceptions from public API because they're only used internally by the CLI applications